src/main/java/org/openhab/automation/jrule/things/JRuleBridgeThing.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.automation.jrule.things;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * The {@link JRuleBridgeThing} thing represents a thing that is connected to a bridge
```

Comment:
JRuleBridgeThing and JRuleStandaloneThing do per JavaDoc the same?

Reply:
I'll update the docs. The idea is that a bridge thing can have methods to list and manipulate sub things

Reply:
Updated